regression/failed-tests-printer.pl

```diff
@@ -6,20 +6,21 @@
 
 my $printed_this_test = 1;
 my $current_test = "";
+my $output_file = "";
 
 while (<LOG>) {
   chomp;
   if (/^Test '(.+)'/) {
     $current_test = $1;
     $printed_this_test = 0;
+  } elsif (/Output:\s+([^\s]+)/) {
+    $output_file = $1;
   } elsif (/\[FAILED\]\s*$/) {
     if(0 == $printed_this_test) {
       $printed_this_test = 1;
       print "\n\n";
       print "Failed test: $current_test\n";
-      my $outf = `sed -n '2p' $current_test/test.desc`;
-      $outf =~ s/\..*$/.out/;
-      system("cat $current_test/$outf");
+      system("cat $current_test/$output_file");
       print "\n\nFailed test.desc lines:\n";
```

Comment:
It's about the literal use of `test.desc` in that `print` statement on line 24. Those files, after all, need not actually be named "test.desc", but could be "test1.desc" etc (your commit says "handle multiple .desc files"). Hence that `print` statement should say, e.g., "Failed test1.desc lines:".